sphinx/source/docs/user_guide/objects.rst

```diff
@@ -186,17 +185,18 @@ properties, for the axis label and major tick labels. These are prefixed ``axis_
 
 Some examples::
 
-    axis.axis_line_color = "red"
-    axis.bounds = (3, 7)
-    axis.major_label_orientation = pi/4
+    ax.axis_line_color = "red"
+    ax.bounds = (3, 7)
+    ax.major_label_orientation = pi/4
 
-Axes for the current plot may be conveniently obtained using the :func:`bokeh.plotting.xaxis`, :func:`bokeh.plotting.yaxis`,
-and :func:`bokeh.plotting.axis` functions. These return collections of axes that can be indexed to retrieve
-individual axes, or can that have attributes set directly on them to update all axes. Examples::
+Axes for a may be conveniently obtained using the ``xaxis``, ``yaxis``,
```

Comment:
`Axes for a may be`...? Is this actually correct wording? Sorry if it is, but does not look common to me...